playground/compound-components/vite.config.ts

```diff
@@ -0,0 +1,14 @@
+import react from '@vitejs/plugin-react'
+import type { UserConfig } from 'vite'
+
+const config: UserConfig = {
+  server: { port: 8907 /* Should be unique */ },
+  mode: 'development',
```

Comment:
```suggestion
```
I guess this isn't needed. It seems it's set in `playground/react/vite.config.ts` as well though.